.github/workflows/acceptance.yml

```diff
@@ -40,13 +40,10 @@ jobs:
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
 
-      - uses: actions/cache@v2
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      - uses: actions/cache@v1 # We are explicitly using v1 due to perf reasons
```

Comment:
If we didn't use the debug logging we could be using v2 and have better perf. No?
<img width="137" alt="image" src="https://user-images.githubusercontent.com/44410/128181406-3aaef721-8b0e-4c71-8f9d-3e2b11692030.png">

Reply:
Yes, but I'd prefer to have debug logging on